diff --git a/node_modules/react-gamepad/lib/Gamepad.js b/node_modules/react-gamepad/lib/Gamepad.js
index 6dfc7ab..4385f9f 100644
--- a/node_modules/react-gamepad/lib/Gamepad.js
+++ b/node_modules/react-gamepad/lib/Gamepad.js
@@ -123,7 +123,7 @@ var Gamepad = function (_React$Component) {
     key: 'updateButton',
     value: function updateButton(buttonName, pressed) {
       if (this.padState.buttons[buttonName] === undefined) {
-        console.error('Unknown button ' + buttonName);
+        // console.error('Unknown button ' + buttonName);
       } else if (this.padState.buttons[buttonName] !== pressed) {
         this.padState.buttons[buttonName] = pressed;
 
diff --git a/node_modules/react-gamepad/src/Gamepad.js b/node_modules/react-gamepad/src/Gamepad.js
index 352acd6..e51bcef 100644
--- a/node_modules/react-gamepad/src/Gamepad.js
+++ b/node_modules/react-gamepad/src/Gamepad.js
@@ -133,7 +133,7 @@ class Gamepad extends React.Component {
 
   updateButton(buttonName, pressed) {
     if (this.padState.buttons[buttonName] === undefined) {
-      console.error(`Unknown button ${buttonName}`)
+      // console.error(`Unknown button ${buttonName}`)
     } else if (this.padState.buttons[buttonName] !== pressed) {
       this.padState.buttons[buttonName] = pressed
 
